modules/solid_mechanics/src/base/SolidMechanicsApp.C

```diff
@@ -146,7 +143,7 @@ SolidMechanicsApp::registerObjects(Factory & factory)
   registerKernel(OutOfPlaneStress);
   registerKernel(StressDivergenceRZ);
   registerKernel(StressDivergenceRSpherical);
-  registerKernel(StressDivergenceTruss);
+  registerKernel(HomogenizationHeatConduction);
```

Comment:
No! You need to rebase and properly resolve your merge conflicts. The `HomogenizationHeatConduction` was moved out of SM last week. Make sure you rebase more often.